src/axom/inlet/SphinxWriter.hpp

```diff
@@ -143,6 +161,17 @@ class SphinxWriter : public Writer
     bool isSelectedElement;
     std::vector<std::vector<std::string>> fieldTable;
     std::vector<std::vector<std::string>> functionTable;
+    /**
+     * \brief Stores a minimal set of data about child containers
+     * Used only for the table of contents in nested "mode"
+     */
+    struct ChildContainerData
+    {
+      std::string name;
+      std::string description;
+      bool isTrivial;

```

Comment:
i know trivial is a semi-used word but I dont like it. this could be my problem though and thats okay.  Can you add a comment here?